tests/meta/workflow/test_microsalt.py

```diff
@@ -0,0 +1,190 @@
+"""Tests for MicroSALT analysis."""
+from typing import List
+
+from cg.apps.tb import TrailblazerAPI
+from cg.meta.workflow.microsalt import MicrosaltAnalysisAPI
+from cg.models.cg_config import CGConfig
+from pathlib import Path
+import logging
+
+
+from cg.store.models import Family
+
+
+def test_qc_check_fail(
+    qc_microsalt_context: CGConfig,
+    microsalt_qc_fail_run_dir_path: Path,
+    microsalt_qc_fail_lims_project: str,
+    microsalt_case_qc_fail: str,
+    caplog,
+    mocker,
+):
+    """QC check for a microsalt case that should fail."""
+    caplog.set_level(logging.INFO)
+    store = qc_microsalt_context.status_db
+    microsalt_api: MicrosaltAnalysisAPI = qc_microsalt_context.meta_apis["analysis_api"]
+
+    # GIVEN a case that is to be stored
+    microsalt_case: Family = store.family(microsalt_case_qc_fail)
+    microsalt_case.samples[0].reads = 1000
+    microsalt_case.samples[1].reads = 1000
+    microsalt_case.samples[2].reads = 1000
+    microsalt_case.samples[3].reads = 1000
+
+    mocker.patch.object(MicrosaltAnalysisAPI, "create_qc_done_file")
```

Comment:
I'm curious, what does this do?

Reply:
I didn't want to create a qc_done file during the tests, so that's why I mocked it to skip this function. I don't want to upload unnecessary files to the repository